Veil-light-wallet-iPhone-sideload.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Sideloading the Veil Light Wallet on iPhone using AltStore</title>
+    <link href="ohcee_guides.css" rel="stylesheet" type="text/css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="UTF-8">
+</head>
+
+<body>
+    <h1>Sideloading the Veil Light Wallet on iPhone using AltStore</h1>
+        <h4>Please install AltStore on your computer and iPhone prior to installation of the Veil Light Wallet.</h4>
+        <h4>Follow the instructions from the link below and add AltStore onto your computer then install AltStore to your iPhone from your computer.</h4>
+        <h4>You can read how to do that here: <a href="https://faq.altstore.io/" target="_blank" rel="noopener noreferrer">AltStore</a></h4>
+
+    <ol>
+        <ul> For this guide I'm using an Apple mac mini M1 machine that is currently running Sonoma 14.1.2</ul>
+        <ul>I'm also using an iPhone 11 running iOS 17.2.</ul>
+        <ul>Things may vary with different operating systems and versions of macOS and iOS.</ul>
+    </ol> 
+
+    <h2>Requirements:</h2>
+        <ol>
+            <li>An iPhone and computer with AltStore installed.</li>
+            <li>A stable internet connection.</li>
+        </ol>
+
+    <h2>Downloading the Veil Light Wallet:</h2>
+        <ol>
+            <li>Visit GitHub to obtain the latest version of the light wallet <a href="https://github.com/steel97/veil_wallet/releases" target="_blank" rel="noopener noreferrer">here</a>.</li>
+            <ul>For iOS it Should look like this:</ul>
+            <figure>
+                <img src="light-wallet-pictures/veil_wallet_ipa.png" alt="veil_wallet_ios_ipa_photo">
+            </figure>   
+            <li>Download the IPA file to your computer. <br>
+                In this demonstration I'm unpacking and moving the IPA file to my iCloud on my mac.</li>
```

Comment:
Release builds isn't packed to zip like in actions tab, so it's better to use just "moving the IPA file" without "unpacking"

Reply:
Fixed it. Thanks.